### DISCUSSION TITLE
Option to drop the -copy flag?

### DISCUSSION THREAD

**Discussion by bdbo** [mifi/lossless-cut#1047] [Ideas]
Great software to cut video's. The issue I ran into was, I wanted exact video cut for guitar exercises on the beat. Now keyframes didn't work out. Dropping the  -copy flag resolved that.

Would be nice if you could  modify the final command on the last screen under "Advanced export options".

Good stuff.

**Comment by mifi** [comment]
Yes, planning on supporting that. See https://github.com/mifi/lossless-cut/issues/372
